attestation-agent/deps/crypto/src/native/aes256gcm.rs

```diff
@@ -10,6 +10,19 @@ use openssl::symm::Cipher;
 
 const TAG_LENGTH: usize = 16;
 
+pub fn decrypt_with_aad(
+    encrypted_data: &[u8],
```

Comment:
@Xynnn007 just curious, any reason that the parameters order are not same for `encrypted_data/ciphertext`, `key`, `iv`, `aad`, `tag` in fn `decrypt_with_aad` and `aes256gcm_decrypt`?

Reply:
Oh. No special reasons. Let me change the order. Thanks for pointing out this

Reply:
Adds a commit for this.